mypy.ini

```diff
@@ -0,0 +1,34 @@
+[mypy]
+python_version=3.7
```

Comment:
Any reason we shouldn't set this to 3.8?

Reply:
Oh maybe that should be left out since it defaults to the python version running `mypy`

https://mypy.readthedocs.io/en/stable/config_file.html#platform-configuration